docs/build/autogenerate.rst

```diff
@@ -738,9 +738,9 @@ generated file path.    Example::
   # format using "black"
   hooks=black
 
-  black.type=console_scripts
-  black.entrypoint=black
-  black.options=-l 79
+  black.type = console_scripts
+  black.entrypoint = black
+  black.options = -l 79 REVISION_SCRIPT_FILENAME
```

Comment:
I think this first example could show the usage without REVISION_SCRIPT_FILENAME, and introduce it below saying something like "this will run `black /path/to/revision.py -l 79`. if we wanted to control the position of the revision path we can use REVISION_SCRIPT_FILENAME" etc

In the other example we can use it, so we show both usages

Reply:
Good idea. I just implemented this in 09c4af8.

Reply:
thanks